File: templates/griffon-javafx-java-templates/templates/subtmpl-artifact/Service.java
package ${project_package};

import griffon.core.GriffonApplication;
import griffon.core.artifact.GriffonService;
import griffon.metadata.ArtifactProviderFor;
import org.codehaus.griffon.runtime.core.artifact.AbstractGriffonService;

import javax.annotation.Nonnull;
import javax.inject.Inject;

@ArtifactProviderFor(GriffonService.class)
public class ${project_class_name}Service extends AbstractGriffonService {
    @Inject
    public ${project_class_name}Service(@Nonnull GriffonApplication application) {
        super(application);
    }
}

File: templates/griffon-javafx-kotlin-templates/templates/subtmpl-artifact/Service.kt
package ${project_package}

import griffon.core.artifact.GriffonService
import org.kordamp.jipsy.ServiceProviderFor
import org.codehaus.griffon.runtime.core.artifact.AbstractGriffonService

@javax.inject.Singleton
@ServiceProviderFor(GriffonService::class)
class ${project_class_name}Service : AbstractGriffonService() {

}
